aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,478 @@
+
+See [Port your code from .NET Framework to .NET Core](/dotnet/core/porting/) for migration information.
+
+## Use the ASP.NET Core shared framework
+
+The [ASP.NET Core 3.0 shared framework](https://natemcmaster.com/blog/2018/08/29/netcore-primitives-2/), contained in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app), no longer requires an explicit `<PackageReference />` element in the project file. The shared framework is automatically referenced when using the `Microsoft.NET.Sdk.Web` SDK in your project file:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+```
+
+## Assemblies removed from the ASP.NET Core shared framework
+
+The most notable assemblies removed from the ASP.NET Core 3.0 shared framework are:
+
+* [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) (Json.NET). To add Json.NET to ASP.NET Core 3.0, see [Add Newtonsoft.Json-based JSON format support](xref:web-api/advanced/formatting#add-newtonsoftjson-based-json-format-support). ASP.NET Core 3.0 introduces `System.Text.Json` for reading and writing JSON. For more information, see [New JSON serialization](#json) in this document.
+* [Entity Framework Core](/ef/core/)
+
+For a complete list of assemblies removed from the shared framework, see [Assemblies being removed from Microsoft.AspNetCore.App 3.0](https://github.com/aspnet/AspNetCore/issues/3755). For more information on the motivation for this change, see [Breaking changes to Microsoft.AspNetCore.App in 3.0](https://github.com/aspnet/Announcements/issues/325) and [A first look at changes coming in ASP.NET Core 3.0](https://devblogs.microsoft.com/aspnet/a-first-look-at-changes-coming-in-asp-net-core-3-0/).
+
+<a name="json"></a>
+
+### New JSON serialization
+
+ASP.NET Core 3.0 includes `System.Text.Json`:
+
+* Reads and writes JSON asynchronously.
+* Is optimized for UTF-8 text.
+* Typically higher performance than `Newtonsoft.Json`.
+
+## gRPC
+
+See <xref:grpc/index>.
+
+## New Razor features
+
+The following list contains new Razor features:
+
+* [@attribute](xref:mvc/views/razor#attribute) &ndash; The `@attribute` directive applies the given attribute to the class of the generated page or view. For example, `@Attribute [Authorize]`.
+* [@implements](xref:mvc/views/razor#implements) &ndash; The `@implements` directive implements an interface for the generated class. For example, `@implements IDisposable`.
+* [@page](xref:mvc/views/razor#page) &ndash; The `@page` directive has different effects depending on the type of the file where it appears. In a *.cshtml* file, `@page` indicates that the file is a Razor Page. In a *.razor* file, `@page` specifies that a Razor component should handle requests directly.
```

Comment:
`@page` isn't new. We just happen to use the same directive in Razor components. I think this bullet can be removed